mythril/laser/ethereum/instructions.py

```diff
@@ -944,16 +944,77 @@ def codecopy_(self, global_state: GlobalState) -> List[GlobalState]:
             global_state.mstate.stack.pop(),
             global_state.mstate.stack.pop(),
         )
+        self._code_copy_helper(
+            code=global_state.environment.code.bytecode,
+            memory_offset=memory_offset,
+            code_offset=code_offset,
+            size=size,
+            op="CODECOPY",
+            global_state=global_state,
+        )
+
+        return [global_state]
+
+    def _get_code_from_address(self, global_state: GlobalState, addr: str) -> str:
+        if addr in global_state.accounts:
```

Comment:
Hmm, not 100% sure but shouldn't it be:
```suggestion
        if addr in global_state.accounts.keys():
```

Reply:
Maybe this is implicitly done by python here

Reply:
Yep